maplibre/src/coords.rs

```diff
@@ -318,7 +375,7 @@ impl WorldTileCoords {
 
     /// Get the tile which is one zoom level lower and contains this one
     pub fn get_parent(&self) -> Option<WorldTileCoords> {
-        if self.z == 0 {
+        if self.z == ZoomLevel::default() {
```

Comment:
Lets add a function to ZoomLevel: `is_root()` which checks if it is `0`

Reply:
Fixed.